initrd/bin/tpmr

```diff
@@ -630,6 +651,7 @@ tpm1_unseal() {
 		-sz "$sealed_size" \
 		-of "$sealed_file" ||
 		die "Unable to read sealed file from TPM NVRAM"
+	# TODO: Cannot log + exit instead of dying!?!
```

Comment:
What's this?

Reply:
Artifacts of quiet modes TODOs that were still in code but not relevant anymore. Fixed by [af59704](https://github.com/linuxboot/heads/pull/1875/commits/af59704bc5c274f247e0c83df194ce0118663dfd)

All other TODOs in code considered relevant per quick review.